packages/host/package.json

```diff
@@ -176,6 +176,7 @@
   },
   "dependencies": {
     "ember-async-data": "^1.0.1",
-    "ember-click-outside": "^6.0.1"
+    "ember-click-outside": "^6.0.1",
+    "jimp": "^0.22.10"
```

Comment:
jimp's unpacked size is 4.96 MB 😬

Reply:
Hmm, I believe we don't need to check whether the icon has a transparent background. Instead, we can simply place a white background behind the icon. If the icon already has a solid background, it will effectively cover the white background. What do you think?

Reply:
Yes, the intention is always use a white background behind any realm icon.